### ISSUE TITLE
Dropdown is not accessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6860]
The Dropdown component is not accessible with Windows High Contrast Mode enabled in either Firefox or Edge.

## Environment

> Operating system

Windows 10

> Browser

Firefox, Edge

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description

![image](https://user-images.githubusercontent.com/40970507/93650150-581e0700-f9d3-11ea-935f-e57940272df8.png)

**Comment by tw15egan** [comment]
Verified fixed on latest

![Screenshot 2021-03-12 121206](https://user-images.githubusercontent.com/11928039/110993483-366f5e00-832c-11eb-87b1-4e3eed576d5e.png)
![Screenshot 2021-03-12 121222](https://user-images.githubusercontent.com/11928039/110993486-3707f480-832c-11eb-9fd2-6b5ad6cf1186.png)
